ui/components/ui/confusable/confusable.stories.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Confusable from './confusable.component';
+
+export default {
+  title: 'Components/UI/Confusable',
+  id: __filename,
+  component: Confusable,
```

Comment:
And and add it here under parameters. You may need to fix the spacing on this suggestion 

```suggestion
  component: Confusable,
  parameters: {
    docs: {
      page: README,
    },
  },
```